instagram/index.html

```diff
@@ -1,9 +1,164 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<html lang="ko">
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Instagram</title>
-  </head>
-  <body></body>
+    <link rel="stylesheet" type="text/css" href="css/reset.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+</head>
+<body>
+<nav class="nav-container">
+    <div class="nav-item-container">
+        <svg class="instagram-icon" width="24" height="24" viewBox="0 0 512 512">
+            <path d="M349.33,69.33a93.62,93.62,0,0,1,93.34,93.34V349.33a93.62,93.62,0,0,1-93.34,93.34H162.67a93.62,93.62,0,0,1-93.34-93.34V162.67a93.62,93.62,0,0,1,93.34-93.34H349.33m0-37.33H162.67C90.8,32,32,90.8,32,162.67V349.33C32,421.2,90.8,480,162.67,480H349.33C421.2,480,480,421.2,480,349.33V162.67C480,90.8,421.2,32,349.33,32Z"></path>
+            <path d="M377.33,162.67a28,28,0,1,1,28-28A27.94,27.94,0,0,1,377.33,162.67Z"></path>
+            <path d="M256,181.33A74.67,74.67,0,1,1,181.33,256,74.75,74.75,0,0,1,256,181.33M256,144A112,112,0,1,0,368,256,112,112,0,0,0,256,144Z"></path>
+        </svg>
+        <hr style="margin-left: 5%; margin-right: 5%">
+        <img class="instagram-logo" src="img/instagram_logo.png" alt="instagram">
+    </div>
```

Comment:
`svg` 쓰는거 너무 좋아요 💯 
다만, `svg` 파일을 분리하고 `img`의 `src` 속성을 이용해 가져온다면 더 좋을것 같아요!

```suggestion
    <div class="nav-item-container">
        <img class="instagram-icon" src="img/instagram-icon.svg" alt="instagram-icon">
        <hr style="margin-left: 5%; margin-right: 5%">
        <img class="instagram-logo" src="img/instagram_logo.png" alt="instagram">
    </div>
```